kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiSessionManager.scala

```diff
@@ -385,14 +385,18 @@ class KyuubiSessionManager private (name: String) extends SessionManager(name) {
       ipAddressLimit: Int,
       userIpAddressLimit: Int,
       userUnlimitedList: Set[String],
-      userLimitedList: Set[String]): Option[SessionLimiter] = {
-    Seq(userLimit, ipAddressLimit, userIpAddressLimit).find(_ > 0).map(_ =>
-      SessionLimiter(
+      userDenyList: Set[String]): Option[SessionLimiter] = {
+    if (Seq(userLimit, ipAddressLimit, userIpAddressLimit).exists(_ > 0) ||
+      Seq(userUnlimitedList, userDenyList).exists(_.nonEmpty)) {
```

Comment:
`userUnlimitedList` should not be used as the basis for whether to create a limiter. Even if it is not empty and there are no other configurations, we do not need to create a limiter.

Reply:
thanks, updated